src/test/java/org/jmxtrans/agent/influxdb/InfluxDbOutputWriterTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2010-2016 the original author or authors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ */
+package org.jmxtrans.agent.influxdb;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jmxtrans.agent.influxdb.InfluxDbOutputWriter.Clock;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+
+/**
+ * @author Kristoffer Erlandsson
+ */
+public class InfluxDbOutputWriterTest {
+
+    private final static long FIXED_TIME = 1234l;
+
+    @Rule
+    public WireMockRule wireMockRule = new WireMockRule(0);
+
+    @Test
+    public void simpleRequest() throws Exception {
+        Map<String, String> s = new HashMap<>();
+        s.put("url", "http://localhost:" + wireMockRule.port());
+        s.put("database", "test-db");
+        stubFor(post(urlPathEqualTo("/write")).willReturn(aResponse().withStatus(200)));

```

Comment:
I love these unit test with HTTP stubs! takes time to write but helpful. Thanks!